### PR TITLE
Vn adding bandit parser to lintly

### PR DESCRIPTION
**Pull request by nandusekarv10**

### CODE REVIEW

File: lintly/parsers.py [hunk 1]
```diff
@@ -117,6 +120,58 @@ def parse_violations(self, output):
...
+        4,
+        5
+      ],
+      "more_info": "https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b410-import-lxml",
+      "test_id": "B410",
+      "test_name": "blacklist"
+    }
+
+    """
+    print("Debug line before entering code")
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Please remove print function.

File: lintly/parsers.py [hunk 2]
```diff
@@ -13,7 +13,8 @@
 class BaseLintParser(object):
 
     def parse_violations(self, output):
-        raise NotImplementedError
+       print(output)
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Please remove print function.

File: deploy.sh [hunk 3]
```diff
@@ -4,4 +4,4 @@ pip install --upgrade setuptools wheel twine
 rm -rf build/
 rm -rf dist/
 python setup.py sdist bdist_wheel
-twine upload dist/*
+#twine upload dist/*
```
Comment (grantmcconnaughey) [inline deploy.sh:7]: This should not be commented out or else deployment won't work anymore.

File: lintly/parsers.py [hunk 4]
```diff
@@ -92,9 +93,11 @@ class PylintJSONParser(BaseLintParser):
     def parse_violations(self, output):
         # Sometimes pylint will output "No config file found, using default configuration".
         # This handles that case by removing that line.
-        if output and output.startswith('No config'):
+        while output.startswith('[main]'):
+          if output and output.startswith('[main]'):
             output = '\n'.join(output.splitlines()[1:])
-
+ 
+        pprint(output)
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Remove

File: lintly/parsers.py [hunk 5]
```diff
@@ -92,9 +93,11 @@ class PylintJSONParser(BaseLintParser):
     def parse_violations(self, output):
         # Sometimes pylint will output "No config file found, using default configuration".
         # This handles that case by removing that line.
-        if output and output.startswith('No config'):
+        while output.startswith('[main]'):
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Why is this change necessary? Also please stick with 4 character indentation.

**Review by grantmcconnaughey — CHANGES_REQUESTED** [review]

**Comment by nandusekarv10** [comment]
I created this my mistake and closing it. It is not done yet. I am trying to write a parser for Bandit as you guys wrote for pylint and other tools. Can you guide me little how can I test it locally when I will make changes?

**Comment by scriptsrc** [comment]
@nandusekarv10  - You should be able to make this a "Draft" PR while you're still working on it.

TODO:
- Remove unnecessary changes. (Should just be the few lines of parser code, the unittest, and the json file used by the test, and an update to the readme -- Looks like there are all kinds of other changes right now that shouldn't be here.)
- Squash commits.
- Possibly check the changes with flake8 and an auto-formatter?

**Pull request closed without merging**